packages/react/src/Prose/Prose.module.css

```diff
@@ -0,0 +1,184 @@
+/* ---------------------------------------------------------- */
+/* Prose component                                            */
+/* ---------------------------------------------------------- */
+/* Table of contents:                                         */
+/* 1. Container                                               */
+/* 2. Element spacing                                         */
+/* 3. Line-length control                                     */
+/* 4. Headings                                                */
+/* 5. Paragraphs                                              */
+/* 6. Lists                                                   */
+/* 7. Inline elements                                         */
+/* 8. Images                                                  */
+/* ---------------------------------------------------------- */
+
+/* ---------------------------------------------------------- */
+/* 1, Container                                               */
+/* ---------------------------------------------------------- */
+
+.Prose {
+  font-family: var(--brand-body-fontFamily);
+  font-size: var(--brand-text-size-300);
+  line-height: var(--brand-text-lineHeight-300);
+  color: var(--brand-color-text-default);
+}
+
+/* ---------------------------------------------------------- */
+/* 2. Element spacing                                         */
+/* ---------------------------------------------------------- */
+
+.Prose * + * {
+  margin-block-start: var(--spacing, 1em);
+}
+
+/* ---------------------------------------------------------- */
+/* 3. Line-length control                                     */
+/* ---------------------------------------------------------- */
+
+.Prose--lineLength {
+  max-width: var(--brand-Prose-lineLength);
+}
+
+/* ---------------------------------------------------------- */
+/* 4. Headings                                                */
+/* ---------------------------------------------------------- */
+
+.Prose h1 {
+  font-weight: var(--brand-text-weight-900);
+  font-size: var(--brand-text-size-900);
+  line-height: var(--brand-text-lineHeight-900);
+  font-family: var(--brand-heading-fontFamily);
+  letter-spacing: var(--brand-text-letterSpacing-900);
+}
+
+.Prose--lineLength h1 {
+  max-width: 22ch;
```

Comment:
I have purposely not included these into the Heading and Text components because it would create a breaking change if these were applied by default. I think this is something we should look into but maybe approach it differently for each of these components.